ra-tls/Makefile

```diff
@@ -19,19 +19,16 @@ ifdef ECDSA
   SGX_DCAP_URI := https://github.com/intel/SGXDataCenterAttestationPrimitives
   SGX_DCAP_COMMIT := bfab1376480f760757738092399d0d99b22f4dfd
   SGX_DCAP ?= SGXDataCenterAttestationPrimitives
-  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include
+  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc \
```

Comment:
Plz split each path component to a single line:

```
 SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc \
<use 2 spaces>-I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc \
...
```

Reply:
In addition, `QuoteVerification/Src/AttestationLibrary/include` doesn't exist already, so plz remove it.